dvtm.c

```diff
@@ -345,6 +346,15 @@ drawbar(void) {
 	attrset(TAG_NORMAL);
 	addstr(layout->symbol);
 
+	if(keys) {
```

Comment:
missing space after if

Reply:
Sure, and I also changed it to display an uppercase mod key, to match vi, so 
`printw("^%c", 'A' - 1 + keys[keycount++]);`

Reply:
Oh, and looks like `if (keys)` condition is no-op. Check https://builds.sr.ht/~kaction/job/449596.

Reply:
This should probably be `if (keys[0])`.

Reply:
It looks like it works as-is, so probably this conditional can be skipped.

Reply:
It is already covered by "while (keycount < MAX_KEYS && keys[keycount])
" loop